src/Nri/Ui/UiIcon/V1.elm

```diff
@@ -2031,26 +2031,56 @@ graduateCap =
+                    , Attributes.stroke "#707070"
+                    , Attributes.strokeWidth "2"
+                    , Attributes.fill "#707070"
```

Comment:
Same problem with fill and stroke color here -- they should change depending on the context, rather than being hardcoded to gray.